doc/cephadm/octopus_gsg.rst

```diff
@@ -0,0 +1,262 @@
+.. _octopus_gsg:
+
+Installing Ceph Octopus
+=======================
+
+This procedure is a step-by-step walkthrough of the installation of a three-node Ceph clutser. If you have never installed Ceph before and you don't know where to begin, begin here.
+
+Removing an old cluster
+-----------------------
+
+If you mess up the installation at any point during the procedure, you can run this two-step procedure to return your system to the beginning state. If you mess up or you lose your place or you just want to start over from the beginning, now you know how.
```

Comment:
```suggestion
If you mess up the installation at any point during the procedure, you can run this two-step procedure
to return your system to the beginning state. Start here if you mess up, lose your place or want
to start over from the beginning.

Reply:
This is a bad example because of the way suggestions work, but this should be a text wrap. 90 characters, please.